docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,119 @@
+            
+            A sample error count vs time graph: ![error-vs-time](https://i.imgur.com/RSH1Szl.png)
+    
+    - "My load test is running, now what?"
+        - If it's a [bounded](https://github.com/mattermost/mattermost-load-test-ng/blob/497554e376ef23d548947bf331c8bdce6ce453d6/docs/faq.md#what-is-a-bounded-load-test) loadtest, it has to be manually stopped with `go run ./cmd/ltctl loadtest stop` [after an hour](https://community.mattermost.com/core/pl/45woi49ru7yrj8r8upzaqhog3a)
+        - If it's an [unbounded](https://github.com/mattermost/mattermost-load-test-ng/blob/497554e376ef23d548947bf331c8bdce6ce453d6/docs/faq.md#what-is-an-unbounded-load-test) loadtest, the load-test will finish with a stdout listing the maximum concurrent users the deployment supports. The load-test status check command will say status as `Done` when it's complete.
+
+#### Analysing load-test results
+
+"My load tests ran successfully, what to make of it?"
+    - In case of unbounded load-tests, when they finish, `go run ./cmd/ltctl loadtest status` would give you a count of maximum concurrent users which is a metric to compare the performance of that version of mattermost-server.
+    - In case of two bounded loadtests with same `MaxConcurrentUsers` count, one can [generate a report](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/compare.md) comparing performance of various server metrics. 
+    <!-- devfix: should we allow comparision of specific store and api functions? -->
+    - In case of both bounded and unbounded loadtests, one can create grafana dashboards to analyze performances of the new features by filtering api metrics relevant to the new handler. Here's an example.
+        ![sample-dashboard-creation](https://i.imgur.com/zzRfh8b.png)
+
+
+#### Creating a PR
+
+After all the code changes,
+ - [Add tests](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/coverage.md#testing) if required. 
+ - `make check-style`
+ - `go get -u github.com/mattermost/mattermost-server/v6@<commit-hash-in-master>` && `go mod tidy`
+ - Make the PR
+
+#### NB:
+
+ - **For seeding the database manually** :
+    - [InstanceConfiguration](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/loadtest_config.md#instanceconfiguration) section would be as minimal as possible to reduce `db init` time.
+    - Message in [Developers:Performance](https://community.mattermost.com/core/channels/developers-performance) for a migration file.
+    - `ssh` into the app machine, and `psql` into the connected database.
+    - Drop all tables, log out of psql. Run the migration, which might take a while. <!-- This might not be necessary, I faced some primary id collision issue while restoring from sql. If someone can confirm it's redundance, we'll delete this instruction.-->
+    - Now, the app service needs to be restarted so the server can run the necessary migrations.
```

Comment:
This is only needed if you are seeding it manually. In an automated comparison, you can just mention the backup file location in https://github.com/mattermost/mattermost-load-test-ng/blob/master/config/comparison.sample.json, and the tool will automatically restore the backup.

Otherwise for a manual restore, we don't need to `psql` into the database. We can just do `zcat <backupfile> | psql <dsn>` after dropping the target DB.

Reply:
This is nice. I didn't discover `comparision` when I was working with this, but I will modify the docs and ask for a re-review.